localization/english/event_localization/nickname/wc_nickname_events_l_english.yml

```diff
@@ -0,0 +1,17 @@
+﻿l_english:
+ WCNIC.0001.t:0 "The Oldest"
+ WCNIC.0001.desc:0 "As a long time has passed since you became the ruler of your tribe, you notice your subjects and the surrounding population has started to refer to you as “the Old Murkeye” due to your long reign and the many years it has been since you have been a tadpole. The noblest of your subjects have suggested the idea of a ceremony to earn this as an official nickname. The ceremony has been prepared and all that is left is for you to do is step on the rock and give the gurgling speech for the army of Murloc raiders in front of you."
```

Comment:
```suggestion
 WCNIC.0001.desc:0 "As a long time has passed since you became the ruler of your tribe, you notice your subjects and the surrounding population has started to refer to you as “the Old Murkeye” due to your long reign and the many years since you have been a tadpole. The noblest of your subjects have suggested the idea of a ceremony to earn this as an official nickname. The ceremony has been prepared and all that is left for you to do is step on the rock and give the gurgling speech for the army of Murloc raiders in front of you."
```

Reply:
```suggestion
 WCNIC.0001.desc:0 "As a long time has passed since you became the ruler of your tribe, you notice your subjects and the surrounding population has started to refer to you as “the Old Murkeye” due to your long reign and the many years it has been since you were but a tadpole. The noblest of your subjects have suggested the idea of a ceremony to earn this as an official nickname. The ceremony has been prepared and all that is left is for you to do is step on the rock and give the gurgling speech for the army of Murloc raiders in front of you."
```

Reply:
I like this one better yeah.

Reply:
done in 23b272b7